src/Microsoft.AspNetCore.Razor.Language/Syntax/LegacySyntaxNodeExtensions.cs

```diff
@@ -0,0 +1,248 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Razor.Language.Legacy;
+
+namespace Microsoft.AspNetCore.Razor.Language.Syntax
```

Comment:
>yntax [](start = 47, length = 5)

Should this be in `.Legacy`? My preference would be that anything we want to **keep forever** should live in `.Syntax` and anything that's transitional should live in `.Legacy`.

Reply:
I'll move it.

---
In reply to: [233618684](https://github.com/aspnet/Razor/pull/2590#discussion_r233618684) [](ancestors = 233618684)